data/json/mapgen/house/house_fence04.json

```diff
@@ -0,0 +1,222 @@
+      "fill_ter": "t_floor",
+      "rows": [
+        "{{{{{{{{{{{,,{{{{{{{{{{{",
+        "{........p----.........{",
+        "{.[.......----.........{",
+        "{..///////0000///////..{",
+        "{../0000000000000000/..{",
+        "{../₸&&₸000000000000/..{",
+        "{..||oo|||o|*o||ooo||..{",
+        "{..|Ty I| yL y|sss??|..{",
+        "{..o@@ I|h    |=====o..{",
+        "{..|    |ff    =H==xo..{",
+        "{..|dd  |ffh   =H==x|..{",
+        "{.%|||+||||  ||=H===|..{",
+        "{.%|w+    b    =l=ER|%.{",
+        "{.^|w+         ====R|^.{",
+        "{.%|||| ||+||  ||  ||%.{",
+        "{..|Dd|+|t_|F 6|y  y|%.{",
+        "{..|    |8_|2 3|Z  io%.{",
+        "{..o @@ |S_|1  |W  io..{",
+        "{..|T@@¤|BB|J5O|q  i|..{",
+        "{..||oo|||o|oo|||o¶||..{",
+        "{................--....{",
+        "{..........[...........{",
+        "{......................{",
+        "{{{{{{{{{{{{{{{{{{{{{{{{"
+      ],
+      "terrain": {
+        "{": "t_chainfence",
+        ",": "t_chaingate_c",
+        "/": "t_railing_h",
+        "_": "t_linoleum_gray",
+        "=": "t_carpet_green",
+        "0": "t_floor_noroof",
+        "H": "t_carpet_green",
+        "E": "t_carpet_green",
+        "R": "t_carpet_green",
+        "?": "t_carpet_green",
+        "l": "t_carpet_green",
+        "s": "t_carpet_green",
+        "x": "t_carpet_green",
+        "w": "t_null",
+        "₸": "t_floor_noroof",
+        "&": "t_floor_noroof"
+      },
+      "furniture": { "&": "f_table", "?": [ "f_indoor_plant", "f_indoor_plant_y" ] },
+      "place_loot": [ { "item": "television", "x": 19, "y": 10 } ]
+    }
+  },
+  {
+    "//": "variant: cat hoarder ",
+    "type": "mapgen",
+    "om_terrain": "house_fence04",
+    "weight": 15,
+    "method": "json",
+    "object": {
+      "palettes": [ "standard_domestic_palette", "standard_domestic_lino_bathroom", "standard_domestic_landscaping_palette" ],
+      "fill_ter": "t_floor",
+      "rows": [
+        "{{{{{{{{{{{,,{{{{{{{{{{{",
+        "{........p----.........{",
+        "{.[.......----.........{",
+        "{..///////0000///////..{",
+        "{../0000000000000000/..{",
+        "{../₸&&₸000000000000/..{",
+        "{..||oo|||o|*o||ooo||..{",
+        "{..|Ty I| yL y|sss??|..{",
+        "{..o@@`I|h`   |=====o..{",
+        "{..|    |ff ` `=H==xo..{",
+        "{..|dd` |ffh   =H==x|..{",
+        "{.%|||}||||  ||=H===|..{",
+        "{.%|w+ `  b   `=l=ER|%.{",
+        "{.^|w+    `  ` ====R|^.{",
+        "{.%||||`||}||  || `||%.{",
+        "{..|Dd|}|t_|F 6|y `y|%.{",
+        "{..| `  |8_|2 3|Z  io%.{",
+        "{..o @@`|S_|1` |W `io..{",
+        "{..|T@@¤|BB|J5O|q` i|..{",
+        "{..||oo|||o|oo|||o¶||..{",
+        "{................--....{",
+        "{....................[.{",
+        "{......................{",
+        "{{{{{{{{{{{{{{{{{{{{{{{{"
+      ],
+      "terrain": {
+        "{": "t_chainfence",
+        ",": "t_chaingate_c",
+        "`": "t_null",
```

Comment:
Looking at the cat hoarder picture further I see that this is messing with the flooring across multiple rooms.  This can fixed relatively easily.  Probably also convert the monster spawns to x,y coordinates or density.

Reply:
Ok I've basically finished this.  I'm thinking longer term it would be best to go ahead and create pet hoarder palettes so they can appear in any house at random similar to the normal hoarder variants.  And then any new houses using the standard general domestic will automatically have that.